src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,83 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SPECIAL_DELIMITER = "!Delimiter@";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        File readFromFile = new File(fromFileName);
+        File writeToFile = new File(toFileName);
+        String[] fileArray = simplificateArray(convertFileToStringArray(readFromFile));
+        if (!fileArray[0].contains("supply")) {
```

Comment:
move index to the class constant

Reply:
not fixed. please move all strings like `",%d"`, `','` to constants